src/collective/listmonk/services/configure.zcml

```diff
@@ -1,3 +1,30 @@
-<configure xmlns="http://namespaces.zope.org/zope">
+<configure
+    xmlns="http://namespaces.zope.org/zope"
+    xmlns:plone="http://namespaces.plone.org/plone"
+    >
+
+  <plone:service
+      method="POST"
+      factory=".subscriptions.CreateSubscription"
+      for="*"
```

Comment:
Shouldn't this be more specific?

Reply:
@ericof site root? It can't be for INewsletter only, because it needs to be possible to subscribe to multiple lists in the same request.

Reply:
Fixed in https://github.com/collective/collective.listmonk/pull/2/commits/82c28e51f71e24cc9b86f4b4ea5de0ee2ca65241